### DISCUSSION TITLE
Why this program needs ffmpeg?

### DISCUSSION THREAD

**Discussion by epiciskandar** [aristocratos/btop#401] [Q&A] [answered]
I've installed btop through brew on my Macbook, noticed that ffmpeg and corresponding indirect dependencies introduced. Since btop is a terminal UI application, I can not understand why it depends on ffmpeg

**Comment by aristocratos** [comment, marked as answer]
It doesn't.

From https://formulae.brew.sh/formula/btop:

> <p>Depends on:</p>
> 
> 
> gcc | 11.3.0 | GNU compiler collection
> -- | -- | --
> 
> 
> Depends on:
> [gcc](https://formulae.brew.sh/formula/gcc) 	11.3.0 	GNU compiler collection
> 
> Depends on when building from source:
> [coreutils](https://formulae.brew.sh/formula/coreutils) 	9.1 	GNU File, Shell, and Text utilities

**Reply by epiciskandar** [reply]
Yes, you are right. I've rechecked dependency by `brew deps --tree --installed btop`, it doesn't have `ffmpeg` dep. And I've tried remove `btop` and all dependencies and reinstalled it, `ffmpeg` don't install this time.

I have `scrcpy` installed, which depends `ffmpeg`, might be some kind of homebrew mechanism triggered `ffmpeg` installing/upgrading during `btop` installation.
